docs/contribute/contributing.md

```diff
@@ -343,7 +342,7 @@ Use their English equivalents instead. Latin abbreviations are sometimes misunde
 Latin  | English equivalent
 ---|---
 e.g.  | for example
-  etc. | and so on. <br/> When you list a clear sequence of elements such as "1, 2, 3, and so on" and "Monday, Tuesday, Wednesday, and so on." Otherwise, rewrite the sentence to replace "etc." with something more descriptive such as "and other output."
+  etc. | and so on. <br> When you list a clear sequence of elements such as "1, 2, 3, and so on" and "Monday, Tuesday, Wednesday, and so on." Otherwise, rewrite the sentence to replace "etc." with something more descriptive such as "and other output."
```

Comment:
This one also needs to be reverted.

All the HTML elements must have a closing tag like `<br/>`, `<img ... />`, unless they would throw Compilation Error.